assets/src/stories-editor/components/with-meta-block-edit.js

```diff
@@ -43,6 +43,7 @@ class MetaBlockEdit extends Component {
 		// If not selected, only change font size if height or width has changed.
 		const checkFontSize = ampFitText && (
 			isSelected ||
+			prevProps.attributes.ampFitText !== ampFitText ||
```

Comment:
🤷‍♂ for consistency reasons I'd prefer to keep it.
I guess we added the `isSelected` check because the content is dynamic? Maybe we should just document that part there.

Reply:
> I guess we added the isSelected check because the content is dynamic? 

Yep, the width and height might change even if not selected. We could maybe improve this comment:
`// If not selected, only change font size if height or width has changed.`

Reply:
Couldn't we then just compare `prevProps.blockContent` with `props.blockContent`? 🤔

Reply:
Hmm, or maybe instead we should remove the `isSelected` and only trigger the font calculation when relevant attributes change? Otherwise, it's triggering for any change, irrelevant included.

Reply:
Yes. `blockContent` would be such a relevant value :)

Reply:
Ha, yes :D I misread your comment, sorry :D

Reply:
Should be fixes in f2e7a5e9c8cf949f32d8839e3c47f4aab025f47c.